toc.md

```diff
@@ -676,7 +678,7 @@ module Workerholic
 end
 ```
 
-To make it work out of the box, Workerholic has default options set up already so you don't need to supply any of the options we mentioned previously. Our default is 25 workers, and the default number of Redis connections is the number of workers + 3, in this case, 28. This is the three additional connections we need for the job scheduler, worker balancer, and the memory trackers.
+To make it work out of the box, we have default options set up already so you don't need to supply anything we mentioned previously. By default, there will be 25 workers and the default number of redis connections is the number of workers + 3, in this case, 28. This is the three additional connections we need for the job scheduler, worker balancer, and the memory tracker.
```

Comment:
I'd move this before the code snippet. Introduce the concept first with explanation first then show the code and explain the code if necessary. I'll leave this comment here but I think this should be done for the all the images/code snippets.

For this particular example I'd also switch from "This is how we did it for Workerholic..." to "a default configuration for a BJP needs to answer the needs of the most common use cases.... This is why for Workerholic we did....."